src/App.tsx

```diff
@@ -1,173 +1,64 @@
-                  <td>
-                    <a href="#/people/jan-frans-van-brussel-1761">
-                      Jan Frans van Brussel
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { PageNavLink } from './components/PageNavLink/PageNavLink';
+
+export const App = () => {
+  return (
+    <>
+      <div data-cy="app">
+        <nav
+          data-cy="nav"
+          className="navbar is-fixed-top has-shadow"
+          role="navigation"
+          aria-label="main navigation"
+        >
+          <div className="container">
+            <div className="navbar-brand">
+              <PageNavLink to="/" text="Home" />
+              <PageNavLink to="people" text="People" />
+            </div>
+          </div>
+        </nav>
+
+        <main className="section">
+          <div className="container">
+            <Routes>
+              <Route
+                path="/"
+                element={<h1 className="title">Home Page</h1>}
+              />
+              <Route path="home" element={<Navigate to="/" replace />} />
+
+              <Route
+                path="people"
+                element={(
+                  <>
+                    <h1 className="title">People Page</h1>
```

Comment:
put it into intext.tsx

Reply:
What do you mean?

Reply:
I mean put all routes to index.tsx and in app.tsx put <Outlet />. But, maybe, it's not necessary